views/default/elements/typography.css.php

```diff
@@ -24,11 +24,11 @@
 	text-decoration: underline;
 }
 
-p {
+.elgg-output p {
```

Comment:
I'd prefer:
```css
.elgg-output p + p {
  margin-top: 15px;
}
```
And then you can remove last-child check, which is no accurate anyway.